EazyE2E/Element/EzRootElement.cs

```diff
@@ -12,22 +12,22 @@ namespace EazyE2E.Element
 {
     public class EzRoot
     {
-        public EzElement RootElement;
+        private readonly EzProcess _process;
+        private EzElement _rootElement;
 
-        public EzRoot(string processName)
-        {
-            RootElement = Set(processName);
-        }
+        public EzElement RootElement => _rootElement ?? (_rootElement = GetRootElement());
 
         public EzRoot(EzProcess process)
         {
-            RootElement = Set(process.ProcessName);
+            _process = process;
         }
-        private EzElement Set(string processName)
+
+        public EzElement GetRootElement()
         {
-            return new EzElement(AutomationElement.RootElement.FindFirst(TreeScope.Children, new PropertyCondition(AutomationElement.NameProperty, processName)));
+            var propertyCondition = new PropertyCondition(AutomationElement.ProcessIdProperty, _process.ProcessId);
+            var element = AutomationElement.RootElement.FindFirst(TreeScope.Subtree, propertyCondition);
```

Comment:
@abduncan May want to do `TreeScope.Children` instead of `TreeScope.Subtree`; because we're starting at the root element, if it can't find the process it will search literally every UI element on the entire computer.  All processes should be treed to the root automation element anyhow.